### DISCUSSION TITLE
[J4dev] Where did the file go Registry.php ?

### DISCUSSION THREAD

**Discussion by korenevskiy** [joomla/joomla-cms#35912] [Q&A] [answered]
Colleagues, Tell me please.
Where did the file go Registry.php ?
Previous file location.
`/libraries/vendor/joomla/registry/src/Registry.php`
But now there is neither a file nor a `/Vendor` folder.
Where did he go `Registry.php` ?

**Comment by laoneo** [comment]
It is still there, but you need to call composer install after you have checked out the repository. More information can be found [here](https://docs.joomla.org/J4.x:Setting_Up_Your_Local_Environment).

**Reply by korenevskiy** [reply]
I wanted to add changes to the file registry.php .
If we add magic methods to this file, we will get a more reactive class.
In which repository is this file located?

**Comment by korenevskiy** [comment, marked as answer]
> It is still there, but you need to call composer install after you have checked out the repository. More information can be found [here](https://docs.joomla.org/J4.x:Setting_Up_Your_Local_Environment).

https://github.com/joomla-framework/registry
It is ?

**Reply by laoneo** [reply]
If you want to make some changes, then open a new pr in this repo https://github.com/joomla-framework/registry.

**Reply by korenevskiy** [reply]
> If you want to make some changes, then open a new pr in this repo https://github.com/joomla-framework/registry.

Thank you for your help.
I couldn't have done it without you.
Please take a look at this PR. https://github.com/joomla-framework/registry/pull/58 
Are there any chances that he will be accepted?

**Reply by laoneo** [reply]
I'm not much involved in the framework stuff, but pretty sure you will get feedback in the pr. Please close this discussion here.

**Reply by korenevskiy** [reply]
> I'm not much involved in the framework stuff, but pretty sure you will get feedback in the pr. Please close this discussion here.

How to do it?
I have no experience deleting a discussion.
I tried but couldn't find the button.

**Reply by brianteeman** [reply]
All you can do is to mark it as answered which you have done
